linux_os/guide/system/kernel_build_config/var_kernel_config_module_sig_key.var

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: 'Keys and certificates for kernel module signing'
+
+description: |-
+    The keys and certificate to use when signing modules during kernel build process.
```

Comment:
```suggestion
    The private key and certificate to use when signing modules during kernel build process.
```

I'd mention something related to the PKCS#11 here as well from the rule description:
```
on systems where the OpenSSL ENGINE_pkcs11
    is functional — a PKCS#11 URI as defined by RFC7512. In the latter case, the PKCS#11 URI should
    reference both a certificate and a private key.
```